### DISCUSSION TITLE
how do I download the War3Evo sound pack?

### DISCUSSION THREAD

**Discussion by ghost** [War3Evo/War3Source-EVO#27] [Q&A] [answered]
how do I download the War3Evo sound pack? link is off
from: https://forums.alliedmods.net/showthread.php?t=284415

**Comment by War3Evo** [comment, marked as answer]
GitHub branch: https://github.com/War3Evo/War3Source-EVO/tree/sound-pack

**Reply by JustZerooo** [reply]
The soundpack was added to the master branch!
